.github/workflows/commit.yaml

```diff
@@ -87,6 +87,13 @@ jobs:
       - name: "Run e2e tests using the `func-e` binary"
         run: make e2e
 
+      - name: "Run e2e tests using the `func-e` binary (CentOS)"
+        if: runner.os == 'Linux'
+        run: docker run -v $HOME/.func-e:/root/.func-e -v $PWD:/func-e --rm ${CENTOS_IMAGE} -c "cd /func-e; make -o ${E2E_FUNC_E_PATH} e2e"
```

Comment:
I thought we could `use: docker://ghcr.io/...` to avoid `docker run -v manual mounts etc...`?

Reply:
oh that would result in the same maybe so nvm 😄

Reply:
yep plus, you can paste this into your laptop and it probably works :D A follow-up change will use the same to test centos+arm64